### PR TITLE
If for task done

### PR DESCRIPTION
**Pull request by kingwangzzang1234**

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,30+1):
+    if i%15==0:
```
Comment (ulgoon) [inline fizzbuzz.py:2]: Can you do it in 1 if statement?

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        print('fizzbuzz')
+    elif i%3==0:
+        print('fizz')
+    elif i%5==0:
+        print('buzz')
+    else:
+        print(i)
```
Comment (ulgoon) [inline fizzbuzz.py:9]: print done! to the last line

**Review by ulgoon — CHANGES_REQUESTED** [review]

File: fizzbuzz.py (hunk 1 above)
Comment (kingwangzzang1234) [reply fizzbuzz.py:2]: Of course!

File: fizzbuzz.py (hunk 2 above)
Comment (kingwangzzang1234) [reply fizzbuzz.py:9]: Gotcha!

Comment (ulgoon) [reply fizzbuzz.py:5]: Well done!!! that's what I want!

**Review by ulgoon — APPROVED** [review]
Well done!

**Pull request merged**